src/background/exceptions.js

```diff
@@ -73,44 +65,27 @@ async function updateFilters() {
       // Negate the filters to make them allow rules
       .map((filter) => `@@${filter.toString()}`);
 
-    for (const filter of filters) {
-      try {
-        const result = (await convert(filter.toString())).rules;
-
-        for (const rule of result) {
-          if (rule.condition.regexFilter) {
-            const { isSupported, reason } =
-              await chrome.declarativeNetRequest.isRegexSupported({
-                regex: rule.condition.regexFilter,
-              });
-            if (!isSupported) {
-              console.error(
-                `Could not add an exception for "${tracker.name}" as filter "${filter.toString()}" is a not supported regexp due to: ${reason}`,
-              );
-              continue;
-            }
-          }
+    if (!filters.length) continue;
 
-          if (domains && domains.length) {
-            if (__PLATFORM__ === 'safari') {
-              rule.condition.domains = domains
-                .map((d) => `*${d}`)
-                .concat(rule.condition.domains || []);
-            } else {
-              rule.condition.initiatorDomains = domains.concat(
-                rule.condition.initiatorDomains || [],
-              );
-            }
-          }
+    const result = await convert(filters);
 
-          rules.push({
-            ...rule,
-            priority: 2000000 + rule.priority,
-          });
+    for (const rule of result.rules) {
+      if (domains && domains.length) {
+        if (__PLATFORM__ === 'safari') {
```

Comment:
shouldn't we use convertToSafariFormat here?

Reply:
It is now in the main convert utility - both checks if regex is supported and converts to Safari format. https://github.com/ghostery/ghostery-extension/pull/2568/files#diff-872e57c910d6cc8e3ca8a688cd4fdc3aa1e864a0092b218a09403859a090424aR35